pkgs/development/python-modules/ffpyplayer/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, pkg-config
+, SDL2
+, ffmpeg-full
+, cython
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "ffpyplayer";
+  version = "4.5.1";
+  src = fetchFromGitHub {
+    owner = "matham";
+    repo = pname;
+    rev = "v${version}";
```

Comment:
When fetching a tag from Git, use `refs/tags/${version}` instead of `${version}`.